src/T2DMap.cpp

```diff
@@ -2281,17 +2280,16 @@ int T2DMap::paintMapInfoContributor(QPainter& painter, int xOffset, int yOffset,
     font.setItalic(properties.isItalic);
 
     int infoHeight = mFontHeight; // Account for first iteration
-    QRect testRect;
     // infoRect has a 10 margin on either side and on top to widget frame.
-    mMapInfoRect = QRect(xOffset, yOffset, width() - 10 - xOffset, infoHeight);
-    testRect = painter.boundingRect(mMapInfoRect.left() + 10, mMapInfoRect.top(), mMapInfoRect.width() - 20, mMapInfoRect.height() - 20, Qt::TextWordWrap | Qt::AlignLeft | Qt::AlignTop, infoText);
+    auto mMapInfoRect = QRect(xOffset, yOffset, width() - xOffset * 2, infoHeight);
```

Comment:
Since `mMapInfoRect` is now a local variable rather than a class member, please consider renaming it to `mapInfoRect` ... :pray:

Reply:
This is draft, wanted to check non local build, the thing is that I cannot reproduce the issue. It exists in 4.12, but not in development branch anymore. so I'd rather close this without merging.